packages/hint/docs/user-guide/index.md

```diff
@@ -86,7 +86,7 @@ or the following if custom:
 Then you have to run the following command to scan a website:
 
 ```bash
-hint https://example.com
+npx hint https://example.com
```

Comment:
Wondering if we should also add `npm run webhint`. If the user has run the prior steps they will already have `hint` installed as a `devDependency` and have the `webhint` script in their `package.json`.

```
npm run webhint -- https://example.com
```

Reply:
IMO, we should explain both options.